script/download-node.js

```diff
@@ -1,75 +1,58 @@
-#!/usr/bin/env node
 var fs = require('fs');
 var mv = require('mv');
 var zlib = require('zlib');
 var path = require('path');
 
-// Use whichever version of the local request helper is available
-// This might be the JavaScript or CoffeeScript version depending
-// on whether this module is being installed explictly or
-// `npm install` is being run at the root of the repository.
-var request = null;
-try {
-  request = require('../lib/request');
-} catch (error) {
-  require('coffee-script').register();
-  request = require('../src/request');
-}
-
-var tar = require('npm/node_modules/tar');
+var tar = require('tar');
 var temp = require('temp');
 
+var request = require('request');
+
+var getInstallNodeVersion = require('./bundled-node-version')
+
 temp.track();
 
 var downloadFileToLocation = function(url, filename, callback) {
   var stream = fs.createWriteStream(filename);
   stream.on('end', callback);
   stream.on('error', callback);
-  request.createReadStream({url: url}, function(requestStream) {
-    requestStream.pipe(stream);
-  });
+  request.get(url).pipe(stream);
 };
 
 var downloadTarballAndExtract = function(url, location, callback) {
   var tempPath = temp.mkdirSync('apm-node-');
   var stream = tar.Extract({
     path: tempPath
   });
-  stream.on('end', callback.bind(this, tempPath));
+  stream.on('end', function() {
+    callback.call(this, tempPath);
+  });
   stream.on('error', callback);
-  request.createReadStream({url: url}, function(requestStream) {
-    requestStream.on('response', function(response) {
-      if (response.statusCode == 404) {
-        console.error('download not found:', url);
-        process.exit(1);
-      }
-      requestStream.pipe(zlib.createGunzip()).pipe(stream);
-    });
+  var requestStream = request.get(url)
+  requestStream.on('response', function(response) {
+    if (response.statusCode == 404) {
+      console.error('download not found:', url);
+      process.exit(1);
+    }
+    requestStream.pipe(zlib.createGunzip()).pipe(stream);
   });
 };
 
 var copyNodeBinToLocation = function(callback, version, targetFilename, fromDirectory) {
   var arch = process.arch === 'ia32' ? 'x86' : process.arch;
   var subDir = "node-" + version + "-" + process.platform + "-" + arch;
-  var fromPath = path.join(fromDirectory, subDir, 'bin', 'node');
-  return mv(fromPath, targetFilename, function(err) {
+  var downloadedNodePath = path.join(fromDirectory, subDir, 'bin', 'node');
+  var targetDir = path.dirname(targetFilename)
```

Comment:
Super minor, but can we 🔥 this line? Seems like we're not using it anywhere in this file.